BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -160,6 +161,30 @@ informative:
 
   DEP-SSLv3: RFC7568
 
+  Boeck2016:
+    author:
+    - ins: H. Boeck
```

Comment:
Can you try to include the umlaut in the guy's name? See https://github.com/cabo/kramdown-rfc2629/issues/105

Reply:
sure!

Reply:
Done in c89df4d